src/connectivity.jl

```diff
@@ -283,6 +283,204 @@ function strongly_connected_components end
     return components
 end
 
+"""
+    strongly_connected_components_kosaraju(g)
+
+Compute the strongly connected components of a directed graph `g` using Kosaraju's Algorithm. 
+(https://en.wikipedia.org/wiki/Kosaraju%27s_algorithm).
+
+Return an array of arrays, each of which is the entire connected component.
+
+### Performance
+Time Complexity : O(|E|+|V|) 
+|V| = Number of vertices
+|E| = Number of edges
+This algorithm requires 2 depth first search passes. Also the algorithm reverses the graph once after one dfs.
+At the end, the graph is reversed again to restore the original graph.
+Also, an iterative version of dfs is written so that this graph doesn't show "stackoverflow" when run on large 
+graphs.
+
+Space Complexity : O(|V|) 
+This memory is used for vectors named color, original, component and components. This space complexity 
+excludes the space required to store the graph.
+
+### Examples
```

Comment:
It is good that you are putting examples here, but try to shorten this list.

Also instead of using `add_edge` in a loop you can  just write `g = SimpleDiGraph(Edge.(edgelist))`.

Reply:
Making use of `g = SimpleDiGraph(Edge.(edgelist))` now.